### ISSUE TITLE
Repos with builds on travis-ci.org take precedence over builds on travis-ci.com

### ISSUE THREAD

**Issue by robbieaverill** [creative-commoners/dashboard#8]
Example: github.com/silverstripe/silverstripe-security-extensions

It was built once a while back on .org, now lives on .com - the dashboard is showing broken master branch only, which is from the .org repo.

Our code pings the .org API which internally resolves to .com when it should, so it must be using .org with priority for this module. We may be able to change to .com endpoint assuming it works the other way too, but I haven't tried it to find out.

**Comment by robbieaverill** [comment]
I've tested using the .com API instead, but it doesn't report correct build statuses for the .org repositories. I thought about building in some logic that uses specific endpoints for certain repositories, but it would also involve handling a separate API token for .com and .org. Instead, I've opted to ask Travis support to delete https://travis-ci.org/silverstripe/silverstripe-security-extensions from .org.
